lib/boundary/hubot/adapters/slack/blocks.ts

```diff
@@ -109,11 +109,15 @@ export const mrkdownWithImgBlock = (
 };
 
 export const headerBlock = (text: string): HeaderBlock => {
+	// Slack HeaderBlock has a 150 character limit
+	const truncatedText =
+		text.length > 150 ? `${text.substring(0, 147)}...` : text;
```

Comment:
We can squeeze in two more characters if we use the unicode ellipses character :)

```suggestion
		text.length > 150 ? `${text.substring(0, 149)}…` : text;
```